src/index.html

```diff
@@ -18,14 +18,14 @@ <h2>Recent Scores</h2>
                         <button class="button-1" type="button">Refresh</button>
                     </div>
                     <table class="table-form">
-
+                
                     </table>
                 </div>
                 <div class="add-your-score">
                     <h2>Add Your Score</h2>
                     <form action="#">
-                        <input type="text" placeholder="Your Name">
-                        <input type="text" placeholder="Your Score">
+                        <input type="text" class="name" placeholder="Your Name">
+                        <input type="text" class="score" placeholder="Your Score">
                         <button class="add-button" type="button">Submit</button>
```

Comment:
- [OPTIONAL] It is a good practice and recommendation to add some validation to your form in order to avoid submission of empty data. Please, always keep this in mind for your future projects that may warrant the use of forms.